src/Templates/DotnetCore/Microsoft.NetMaui.CSharp.ProjectTemplates/MauiMobile.vstemplate

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
+  <TemplateData>
+    <Name>.NET MAUI App</Name>
+    <Description Package="{b3bae735-386c-4030-8329-ef48eeda4036}" ID="4655" />
+    <Icon Package="{b3bae735-386c-4030-8329-ef48eeda4036}" ID="4600" />
```

Comment:
Is it okay for all three of these to have the same description?

Reply:
Yes, these templates don't actually show up in the new project wizard, they just point to templates that get installed under the .NET SDK directory.  So, description, Icon, etc. values don't really matter.